tools/hangar/__snapshots__/test_corpus/valid/for_loop.test.w_compile_tf-aws.md

```diff
@@ -286,96 +286,96 @@ class $Root extends $stdlib.std.Resource {
       {((cond) => {if (!cond) throw new Error("assertion failed: postContinueHits == 0")})((((a,b) => { try { return require('assert').deepStrictEqual(a,b) === undefined; } catch { return false; } })(postContinueHits,0)))};
     }
     {((cond) => {if (!cond) throw new Error("assertion failed: j == 3")})((((a,b) => { try { return require('assert').deepStrictEqual(a,b) === undefined; } catch { return false; } })(j,3)))};
-    {console.log("---\nfor x in 0..0 { ... }")};
+    {console.log("---\nfor x in 0..0 \{ ... }")};
```

Comment:
Shouldn't the `\` not be here here?

Reply:
good point, in the JS output we can remove this escaping, even though it doesn't matter much as far as I can tell.